style.css

```diff
@@ -559,8 +557,233 @@ button:hover {
   .foot {
     display: none;
   }
+
+  /* PopUp */
+  .modal {
+    display: none;
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    overflow: auto;
+    background: #c1c7d0;
+    z-index: 1;
+  }
+
+  .pop-up {
+    width: 60%;
+    margin: 30px auto;
+    padding: 5px 20px;
+    background-color: #fff;
+    top: 60px;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid #ebecf0;
+    border-radius: 12px;
+    box-shadow: 2px 2px 8px #ebecf0;
```

Comment:
- [ ] Good job working on the modal pop-up, but on the mobile view the action buttons are too far apart from each other which is different from what we have in the original design. Please kindly fix this.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/232203141-d26d70d6-dded-4516-88a2-28d145b9775a.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/232203184-729f5403-e3f8-4132-8267-028208ac96e2.png)